src/main/resources/com/mathworks/ci/MatlabBuilder/runMatlabTests.m

```diff
@@ -12,10 +13,14 @@
 import('matlab.unittest.TestSuite');
 
 %Create test suite for tests folder
-suite = testsuite(pwd,'IncludeSubfolders',true);
+if verLessThan('matlab',BASE_VERSION_TESTSUITE_SUPPORT)
```

Comment:
Would it make it easier to read if this logic was refactored out to a separate local function. 

the code in runMATLABTests would simply become

suite = gettestsuite(); 

And the special case logic related to selection of the correct testsuite as per release will be in a its own function.

Reply:
Done.